CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 - Migrate to Gradle Version Catalog  
 - Move logic for deciding the initial screen to the Mobius loop in `TheActivity`
 - Update OverdueListChanges feature flag to support `ContactPatientSheet` changes
+- Set Sri Lanka display name in country selection screen  
```

Comment:
Shouldn't the Sri Lanka be a pointer under `Add support for Sri Lanka`?

Reply:
It's not a public change yet. We haven't enabled the support for Sri Lanka right now.

Reply:
Right, and the changelog says, it's in progress. 
Wouldn't this be like :

```suggestion
- in progres Add support for Sri Lanka
   - Set Sri Lanka display name in country selection screen ```

Reply:
Nope, it's just an internal change.

Reply:
So basically even after we add this country support, users will not be able to see it yet. The server has to start sending us the manifest with the new country. So most of these changes will be internal.

Reply:
So Add support for Sri Lanka is also added as internal change, right? 
I know it's internal, just meant to say, this should be a pointer inside the feature. Like we started doing.

Reply:
We are only doing that for features that require more explanation or have a vague changelog. For example things like overdue list improvements, we have added what the improvements are. In this situation we are adding support for a new country, I don't think it needs any more explanation. Once the server has finished the Sri Lanka support work, we have to coordinate the release changelog.

Reply:
If you look at our existing changelog on master, you can see we are not adding points for every new feature. We are adding them in internal and we are only adding one-liners most of the time that are user-facing and are useful for them.

Reply:
I have updated the changelog. I have kept the changes in internal but moved the in-progress of entry from internal to changes.